### DISCUSSION TITLE
Hosting scripts

### DISCUSSION THREAD

**Discussion by vuthanhdatt** [vibalcam/ticktick-gcalendar-py#1] [Q&A] [answered]
Do you hosting this script to sync automatically instead of run mannually

**Comment by vibalcam** [comment, marked as answer]
I have this script set up on a raspberry pi so it runs automatically every 30 minutes (I use cron for that). You can do the same and have it in your own computer set up so it runs periodically.

I am not sure if that answers your question.

**Reply by vuthanhdatt** [reply]
Actually I want live update but periodically update also a good idea, I'll consider using Github Action. Thanks for your answer.

**Reply by vibalcam** [reply]
The script does not support live updates right now, but if you really want that functionality, you could use tools such as IFTTT to run the script when a task is updated in google calendar or TickTick.

However, I believe this is not necessary. In practice, you can get the same functionality by using a very small refresh time, such as 1 minute.

I hope this helps.
